### DISCUSSION TITLE
Welcome to ha-xsense-component_test Discussions!

### DISCUSSION THREAD

**Discussion by Jarnsen** [Jarnsen/ha-xsense-component_test#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MusicmanP3** [comment]
Hi
Is it possible to have home security armed status integrated? (Disarm, Home and Away)
Would be essential for my automations in HA.

**Reply by Jarnsen** [reply]
If Its in the App, sure. But i Don‘t know when it will be implemented

**Reply by MusicmanP3** [reply]
Thank you! I will be waiting! I'm hoping sooner than later. ;)

Yes it is in the app.  
https://www.x-sense.com/cdn/shop/files/2_6ed4261c-fca0-43b2-8ab1-fd8f143df289_800x800.jpg?v=1728472143

**Reply by MusicmanP3** [reply]
Hello
Besides from what I wrote earlier, can you also implement cameras to the integration?

But my last post is more important. Any news about that? :
(Is it possible to have home security armed status integrated? (Disarm, Home and Away))

**Reply by MusicmanP3** [reply]
Any news with this one? (Status: Disarm, Home(night) and Away)

**Reply by fabi9276** [reply]
I would love to have the integration for the modes (Home/away/off) of the basestation, too. Let me know, if i can help with testing or something else!

**Comment by ItzChriZz** [comment]
Hi, can you make an Button to reset the Smoke Alarm after it has been triggered by e.g. a Fire?

**Comment by Jensekin** [comment]
Hi, Could you please add the smoke detectors XP02S-MR? I have them in the App and I can also see them in HA but without any data (Battery level and so on).

**Comment by Jarnsen** [comment]
Theo is working on some things. Camera an such things. If you would to integrate some sensors, please Share with theo.

But it Theo needs time, he is very busy!

**Comment by DannyMacAllister** [comment]
Hi is it possible to integrate a entity to activate/trigger the Alarm from the XS0B-MR to use them in a alarmsystem as speakers?

**Reply by Tweak90** [reply]
> Hi is it possible to integrate a entity to activate/trigger the Alarm from the XS0B-MR to use them in a alarmsystem as speakers?

Heyho, first of all thanks for your great work @Jarnsen!

I have the same Question as @DannyMacAllister, I'm using multiple xs01-m and XH02-M and would love to use them as alarmsystem-speakers.

**Reply by Jarnsen** [reply]
Currently this is not possible. Theo is working on several things. At the moment the app is being simulated, and since there's nothing in the app to directly trigger a real alarm, this isn't possible right now - at most you can do a test alarm if your sensor supports that in the app. There are thoughts about simulating a sensor and potentially triggering an alarm that way, but these are just ideas at this point. There's nothing more concrete yet. Theo hasn't been in touch for a while, I think he's really busy at the moment.

**Comment by gp-peri** [comment]
Hello, thank you very much for your work. Are there plans to integrate the hygrometer-thermometer (STH0A)?
